.circleci/config.yml

```diff
@@ -46,7 +48,9 @@ commands:
           when: always
       - run:
           name: 'Solidus <<parameters.solidus_branch>>:  Install sandbox with generated starter frontend'
-          command: bin/sandbox --seed=false --sample=false
+          command: |
+            <<#parameters.coverage>>export COVERAGE=true<</parameters.coverage>>
```

Comment:
Do we need coverage when creating the sandbox application? :thinking:

Reply:
In theory we would like to track usage of `template.rb` too, but I think it's still not fully working and will need additional passes once we get `codecov` going with a baseline on the main branch

Reply:
Not sure if getting that. When we run `bin/sandbox`, we're not running any spec file, right? How can `COVERAGE` make a difference there? :thinking:

Reply:
Coverage tracking works independently of having specs being run, just tracks which lines are executed during the life of a program. That means that if `template.rb` is executed after `Coverage.start` we'll get a list of lines for it in `Coverage.result`.

Since our coverage setup is "activated" by the `COVERAGE` env var we what it set when we install solidus and end up running the `template.rb` file. 

I hope it's clearer, but happy to jump on a call and go into the details.